### PR TITLE
[UT] Support for running additional QueryDump test frameworks

### PR DESCRIPTION
**Pull request by HangyuanLiu**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [x] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

File: fe/fe-core/src/test/java/com/starrocks/sql/dump/QueryDumpCase.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.sql.dump;
+
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+
+public class QueryDumpCase {
+
+    @JacksonXmlProperty(localName = "name")
```
[the remark this answers is not in the document]
Comment (HangyuanLiu) [reply fe/fe-core/src/test/java/com/starrocks/sql/dump/QueryDumpCase.java]: done

**Review by kangkaisen — DISMISSED** [review]

**Review by Seaven — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by HangyuanLiu** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by kangkaisen — APPROVED** [review]

**Review by Seaven — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**